doc/spec.md

```diff
@@ -34,4 +34,14 @@ Pokud potřebujete nějakou hodnotu nastavit během vývoje, můžete ji uložit
 * `ECOMAIL_API_KEY` je klíč k Ecomailu, používáme ho pro registraci nových zájemců o newsletter
 * `SLACK_CLIENT_ID` a `SLACK_CLIENT_SECRET` slouží pro přihlašování uživatelů přes funkci _Sign in With Slack_
 * `NEXTAUTH_URL` je URL, na které se má přesměrovat OAuth flow při přihlašování k webu
-* `NEXTAUTH_SECRET` je tajemství používané pro podepisování sessions
\ No newline at end of file
+* `NEXTAUTH_SECRET` je tajemství používané pro podepisování sessions
+
+# Uživatelské účty
+
+Základní data o uživatelích máme rozdělená do dvou propojených tabulek: Tabulka _User Profiles_ obsahuje data přímo spravovaná uživatelem (například seznam jeho kompetencí), tabulka _Slack Users_ obsahuje data získaná ze Slacku (například profilový obrázek).
+
+## Založení účtu
+
+1. Uživatel vyplní onboardovací formulář na adrese join.cesko.digital, kde zadá základní údaje, zejména e-mail. Po odeslání uložíme do tabulky _User Profiles_ nový uživatelský profil ve stavu `unconfirmed`. (TBD: Co když už daný e-mail v databázi je?)
```

Comment:
> TBD: Co když už daný e-mail v databázi je?
Tady bychom asi měli vrátit nějakou chybu a další workflow neprovádět?

Reply:
Teď jsem koukal na `/api/user_profiles` (kam se odesílá onboardovací formulář) a už to tam kontrolujeme, takže dobrý. Další zajímavá otázka ovšem: Co když už je příslušný mail ve Slacku, ale neprošel onboardingem?

Reply:
Tak to už je zapeklitější otázka :) Asi platí, že duplicitní emaily nechceme za žádnou cenu, takže vrátit taky nějakou chybu, s odkazem na nějaký "support" email?